README.md

```diff
@@ -82,6 +82,30 @@ All of the arguments are optional:
 
 `--parsers`, `--detectors` and `--specials`: These arguments are for advanced usage. They provide an easy way to customize the file parser and dependency detection. Check [the pluggable design document](https://github.com/depcheck/depcheck/blob/master/doc/pluggable-design.md) for more information.
 
+## Usage with a configuration file
+
+Depcheck can be used with an rc configuration file. In order to do so, create a .depcheckrc file in your project's package.json folder, and set the CLI keys in YAML, JSON, and Javascript formats.
+For example, the CLI arguments `--ignores="eslint,babel-*" --skip-missing=true` would turn into:
+
+***.depcheckrc***
+```
+ignores: ["eslint", "babel-*"]
+skip-missing: true
+```
+
+**Important:** if provided CLI arguments conflict with configuration file ones, the CLI ones will take precedence over the rc file ones.
+
+The rc configuration file can also contain the following extensions: `.json`, `.yaml`, `.yml`, `.js`, `.config.js`.
+If the `.js` or `.config.js` extension are used, then the file should export a Javascript object:
```

Comment:
Is there a way to tell cosmiconfig not to load javascript files? I really want the configuration to be static

Reply:
Yeah, will be pushed with my next change.